tls/s2n_x509_validator.h

```diff
@@ -62,6 +62,32 @@ struct s2n_x509_validator {
     int state;
 };
 
+struct s2n_x509_cert {
+    X509 *cert;
+};
```

Comment:
This struct doesn't make a lot of sense unless you know that it's intended as a public wrapper. You might want to put a TODO on all these APIs or something else to indicate they're part of an unreleased feature.

Reply:
I added a comment for this.